website/content/docs/commands/pki/health-check.mdx

```diff
@@ -217,7 +238,11 @@ This health check checks each role to see whether the role allows localhost base
 
  - `allowed_roles` `(list: nil)` - an allow-list of roles to ignore.
 
-This health check checks each role to see whether or not it allows wildcard issuance with glob domains as well - these two interact and can result in nested wildcards and other quirks. It is strongly suggested that roles either allow globs or or allow wildcards, but not both - this will be a critical warning. If both behaviors are required, we'd suggest splitting it into two roles.
+This health check checks each role to see whether or not it allows wildcard issuance with glob domains as well - these two interact and can result in nested wildcards and other quirks.
+
+**Remediation Steps**:
+
+It is strongly suggested that roles either set `allow_glob_domains` or `allow_wildcard_certificates`, but not both to true. If both behaviors are required, we'd suggest splitting it into two roles unless required for all SANs on the certificate. Otherwise, add this role to `allowed_roles` to ignore future warnings about this role.
```

Comment:
```suggestion
1. Split any role that need both of `allow_glob_domains` and `allow_wildcard_certificates` to be true into two roles.
1. Continue splitting roles until both of the following are true for all roles:
    - The role has `allow_glob_domains` **or** `allow_wildcard_certificates`, but
      not both.
    - Roles with `allow_glob_domains` **and** `allow_wildcard_certificates` are
      the only roles required for **all** SANs on the certificate.
1. Add the roles that allow glob domains and wildcards to `allowed_roles` so
   Vault ignores them in future checks.
```

I'm assuming that making sure only one of the two values is true is the only appropriate remediation. If so, mentioning that it's strongly recommended isn't necessary. The fact that we're providing it as a remediation already implies that ;)